MSMetaEnhancer/libs/utils/Job.py

```diff
@@ -26,8 +26,7 @@ def validate(self, converters, metadata):
             raise ConversionNotSupported(f'Conversion ({self.converter}) {self.source} -> {self.target}: '
                                          f'is not supported')
         elif data is None:
-            raise SourceAttributeNotAvailable(f'Conversion ({self.converter}) {self.source} -> {self.target}: '
-                                              f'Attribute {self.source} missing in given metadata.')
+            raise SourceAttributeNotAvailable(f'{self}:\n Attribute {self.source} missing in given metadata.')
```

Comment:
Does this use the __str__ function to represent self or the __repr__ function?

Reply:
Here `__str__` method is implicitly called.